internal/policy/verify.go

```diff
@@ -683,105 +608,59 @@ func verifyEntry(ctx context.Context, repo *git.Repository, policy *State, attes
 		return fmt.Errorf("verifying file namespace policies failed, %w", ErrUnauthorizedSignature)
 	}
 
-	return nil
-}
-
-func verifyTagEntry(ctx context.Context, repo *git.Repository, policy *State, entry *rsl.ReferenceEntry) error {
```

Comment:
I think this is going to overload the verification of a branch entry and a tag entry. I suggest rolling this change back. IMO, verifyEntry and verifyTagEntry ought to be separate.

We need to have a different conversation about the `verify-tag` (and `verify-commit`) user facing endpoints, but here, let's keep them separate so we can better reason about it.

Reply:
Also, doesn't this break with `getCommits` for a tag? When we load the commits modified in the case of this entry to find modified files?

Reply:
I have rolled back the last commit